### PR TITLE
Dark Theme for Klasa Website

### PR DESCRIPTION
**Pull request by AdityaTD**
## Added Dark Theme (Toggleable)
Requires a bit of cleanup, otherwise, it's almost ready to use in production as is.

Dark Theme:
image

Light Theme:
image

----

Dark Theme Docs:
image

Light Theme Docs:
image

----

Toggling Gif:
image

### CODE REVIEW

**Comment by UnseenFaith** [comment]
Pictures of actual documentation por favor.

File: src/App.vue [hunk 1]
```diff
@@ -16,111 +16,102 @@ export default {
 		AppNavbar,
 		AppFooter
 	},
+	data() {
+		return {
+			dark: JSON.parse(localStorage.getItem('dark'))
```
Comment (MrJacz) [inline src/App.vue]: this should be computed and not in data

File: src/App.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-	<div class="site">
+	<div class="light site">
```
Comment (MrJacz) [inline src/App.vue]: if you make the computed prob called theme that returns the class name you can do `:class="theme"` so it looks like
```html
<div class="site" :class="theme">
```

File: src/App.vue [hunk 3]
```diff
@@ -16,111 +16,102 @@ export default {
...
+	},
 	methods: {
 		invite() {
 			const popup = window.open('about:blank', '', 'toolbar=no,scrollbars=yes,resizable=yes,width=498,height=666');
 			fetch('https://api.klasa.me/invite')
 				.then(res => res.json())
 				.then(({ invite }) => { popup.location = invite; });
+		},
+		setTheme() {
+			if (this.dark) this.$el.classList.replace('light', 'dark');
```
Comment (MrJacz) [inline src/App.vue]: We're using Vue so we can actually use directives as shown above

File: src/App.vue (hunk 1 above)
Comment (bdistin) [reply src/App.vue]: this can't be computed, as we need to actually change state for the site to change it's look.

**Pull request merged**